src/Notifications/RefreshToken/AbstractRefreshTokenNotification.php

```diff
@@ -41,4 +43,34 @@ abstract class AbstractRefreshTokenNotification extends AbstractNotification
     {
         return true;
     }
+
+    /**
+     * @var array
+     */
+    protected $tags = ['refresh_token'];
+
+    public $user_name;
+
+    public $image;
+
+    public $main_character;
+
+    public $corporation;
+
+    public $refresh_token;
+
+    public function __construct(RefreshToken $refresh_token)
```

Comment:
Why don't init `refresh_token`, `user_name`, `main_character`, ``corporation` into the parent constructor ?

Reply:
because partent constructor is used by any notification. F.e. in `kill_mail` you don't have a `refresh_token`.